src/config/dynamic-theme-fixes.config

```diff
@@ -604,6 +604,16 @@ INVERT
 
 ================================
 
+alt.hololive.tv
+
+INVERT
+video[src*="bg.mp4"]
```

Comment:
Or there is being something appended to this otherwise you can also replace the `*` to `$`

Reply:
There's something appended to it `?20210215165645` (Seems like a upload date or something `? 2021 02 15 165645`)

Reply:
Got it!